queue/amqp.go

```diff
@@ -464,5 +479,28 @@ func fromDelivery(d *amqp.Delivery) (*Job, error) {
 	j.tag = d.DeliveryTag
 	j.raw = d.Body
 
+	if retries, ok := d.Headers[retriesHeader]; ok {
+		retries, ok := retries.(string)
```

Comment:
Can keep `int` for retries originally instead of converting it back and forth from/to string/int? (Especially that Headers is an instance of `type Table map[string]interface{}`

Reply:
I did it in that way at first, but it seems that you can save `int` as values. Checking it again you can use integers with defined length so i'm changing it to use `int32`.